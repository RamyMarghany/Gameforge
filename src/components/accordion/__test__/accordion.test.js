import React from 'react';
import ReactDOM from 'react-dom'
import Accordion from './../index'
import "@testing-library/jest-dom"; 
import renderer from 'react-test-renderer'

it('render without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Accordion></Accordion>, div)
})


it('matches snapshots', ()=>{
    const tree = renderer.create(<Accordion/>).toJSON()
    expect(tree).toMatchSnapshot();
})