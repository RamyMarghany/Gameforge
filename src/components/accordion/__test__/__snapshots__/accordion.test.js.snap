// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<div
  className="accordion-container"
  data-testid="accordion"
>
  <div
    className="stylecss__AccordionTitle-sc-1jkeuws-0 kZrPQQ accordion-title"
    onClick={[Function]}
  />
  <div
    className="stylecss__AccordionBody-sc-1jkeuws-1 drDpFH accordion-body"
  />
</div>
`;
